EventBusAnnotationProcessor/src/org/greenrobot/eventbus/annotationprocessor/EventBusAnnotationProcessor.java

```diff
@@ -81,8 +81,8 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
         try {
             String index = processingEnv.getOptions().get(OPTION_EVENT_BUS_INDEX);
             if (index == null) {
-                messager.printMessage(Diagnostic.Kind.ERROR, "No option " + OPTION_EVENT_BUS_INDEX +
-                        " passed to annotation processor");
+//                messager.printMessage(Diagnostic.Kind.ERROR, "No option " + OPTION_EVENT_BUS_INDEX +
+//                        " passed to annotation processor");
```

Comment:
Nit: Do we want to update this message or just leave a comment here?

Reply:
I wanted to keep the code as close to original as possible. Calling `messager.printMessage(Diagnostic.Kind.ERROR)` is what compiler interprets as the error and breaks compilation. So we do not want the message but the code should still stay there.